src/test/config.js

```diff
@@ -41,6 +44,7 @@ describe('embark.Config', function () {
     it('should convert Ether units', function () {
       let expectedConfig = {
         "enabled": true,
+        "ethereumClientName": "geth",
```

Comment:
It would be really cool if we could just do

```
import { GETH_CLIENT_NAME } from '@embark/core';
```

And use that. If we make `geth` a constant somewhere in our source, we can at least require it from there.